client/hudwidget.cpp

```diff
@@ -1834,17 +1837,24 @@ void hud_unit_combat::paintEvent(QPaintEvent *event)
   p.setFont(f);
   p.setPen(QColor(Qt::white));
   if (def_veteran) {
-    p.drawText(right,
-               Qt::AlignHCenter | Qt::AlignJustify | Qt::AlignAbsolute,
-               QStringLiteral("*"));
+    p.drawText(right, Qt::AlignBottom | Qt::AlignRight | Qt::AlignAbsolute,
+               QStringLiteral("*Vet*"));
   }
   if (att_veteran) {
-    p.drawText(left, Qt::AlignHCenter | Qt::AlignJustify | Qt::AlignAbsolute,
-               QStringLiteral("*"));
+    p.drawText(left, Qt::AlignBottom | Qt::AlignRight | Qt::AlignAbsolute,
+               QStringLiteral("*Vet*"));
   }
   p.drawText(left, Qt::AlignHorizontal_Mask, ahploss);
   p.drawImage(right, dimg);
   p.drawText(right, Qt::AlignHorizontal_Mask, dhploss);
+  if (def_win) {
+    p.drawText(right, Qt::AlignBottom | Qt::AlignLeft | Qt::AlignAbsolute,
+               QStringLiteral("Win"));
+  }
+  if (att_win) {
+    p.drawText(left, Qt::AlignBottom | Qt::AlignLeft | Qt::AlignAbsolute,
+               QStringLiteral("Win"));
+  }
```

Comment:
Can both `def_win` and `att_win` be true simultaneously? Maybe with `Combat_Rounds` or bombarding?

Reply:
I wasn't sure so I went with a variable for both and added them to the `if (defender_hp <= 0)` block